include/kodlab_mjbots_sdk/imu_data.h

```diff
@@ -22,6 +22,16 @@ namespace kodlab
 
 /**
  * @brief IMUData object storing inertial data.
+ * @note The body, world, and modified world frames are referred to throughout
+ *       this class' documentation. The body frame is fixed to the robot body
+ *       and offset from the IMU/Pi3Hat frame based on the Mjbots control loop
+ *       options `imu_mounting_deg` parameters. The world frame describes a
+ *       fixed inertial frame which is defined at robot startup. This frame
+ *       always has \f$\hat{z}\f$ pointing in the direction of gravity. The
```

Comment:
This should not mention MJBots if we are trying to keep things modular. #29 is a useful place to start